examples/webgl_loader_md2.html

```diff
@@ -152,7 +152,7 @@
 
 				} );
 
-				gui.add( playbackConfig, 'wireframe', false ).onChange( function () {
+				gui.add( playbackConfig, 'wireframe' ).onChange( function () {
```

Comment:
`gui.add` doesn't do anything with the third parameter if the property value is a boolean.